components/Common/Search/States/index.module.css

```
@@ -0,0 +1,215 @@
+.searchBoxModalContainer {
+  @apply fixed
+    inset-0
+    z-50
+    flex
+    items-center
+    justify-center
+    bg-neutral-900
+    bg-opacity-90
+    dark:bg-neutral-900
+    dark:bg-opacity-90;
+}
+
+.searchBoxModalPanel {
+  @apply fixed
+    h-screen
+    w-full
+    bg-neutral-100
+    p-2
+    dark:bg-neutral-950
+    md:top-60
+    md:h-[450px]
+    md:max-w-3xl
+    md:rounded-xl
+    md:shadow-lg;
+}
+
+.searchBoxInnerPanel {
+  @apply pt-12
+  text-neutral-800
+  dark:text-neutral-400
+    md:p-2;
+}
+
+.searchBoxMagnifyingGlassIcon {
+  @apply absolute
+  top-[10px]
+  hidden
+  h-6
+  w-6
+  md:block;
+}
+
+.searchBoxBackIconContainer {
+  @apply block
+  md:hidden;
+}
+
+.searchBoxBackIcon {
+  @apply absolute
+  top-[7px]
+  block
+  h-6
+  w-6
```

```suggestion
  size-6
```

[tailwind docs](https://tailwindcss.com/docs/size)